### PR TITLE
[Fix-2248][admin][metadata] Modification only supports clickhouse driver version 0.2.6

### PR DESCRIPTION
**Pull request by LiuHao0606**
1.Upgrade the clickhouse driver to version 0.6.2
2.When modifying the connection data source, only the clickhouse driver version 0.2.6 is supported
3.The driver class for ClickHouse has been changed from ru.yandex.clickhouse.ClickHouseDriver to com.clickhouse.jdbc.ClickHouseDriver

### CODE REVIEW

File: dinky-admin/pom.xml [hunk 1]
```diff
@@ -345,8 +345,14 @@
             <groupId>org.dinky</groupId>
             <artifactId>dinky-alert-sms</artifactId>
         </dependency>
+<!--        本地调式时打开，如果需要测试其他数据源，请添加其他数据源依赖在此处-->
```
Comment (Zzm0809) [inline dinky-admin/pom.xml]: 无需增加此处, 在 dinky-core/pom.xml 已经可以支持本地调试场景

File: dinky-admin/src/main/java/org/dinky/controller/DataSourceController.java [hunk 2]
```diff
@@ -195,7 +195,9 @@ public Result<List<DataBase>> listEnabledAll() {
             dataType = "DataBaseDTO",
             paramType = "body",
             dataTypeClass = DataBaseDTO.class)
-    public Result<Void> testConnect(@RequestBody DataBaseDTO dataBaseDTO) {
+    public Result<Void> testConnect(@RequestBody DataBaseDTO dataBaseDTO) throws ClassNotFoundException {
+        //测试驱动是否有正常加载
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/controller/DataSourceController.java]: Driver 中已经try catch 了 此处无需再次 throws

File: dinky-admin/src/main/java/org/dinky/controller/DataSourceController.java [hunk 3]
```diff
@@ -195,7 +195,9 @@ public Result<List<DataBase>> listEnabledAll() {
...
+        //        Class.forName("org.dinky.metadata.driver.ClickHouseDriver");
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/controller/DataSourceController.java]: 删除此处

File: dinky-metadata/dinky-metadata-base/pom.xml [hunk 4]
```diff
@@ -92,6 +92,23 @@
...
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.dinky</groupId>
+            <artifactId>dinky-common</artifactId>
+        </dependency>
+
+        <!--本地调试开启以下两项-->
+<!--        <dependency>-->
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-base/pom.xml]: 删除此处,具体数据源模块中已包含

File: dinky-metadata/dinky-metadata-clickhouse/pom.xml [hunk 5]
```diff
@@ -41,8 +41,9 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>ru.yandex.clickhouse</groupId>
+            <groupId>com.clickhouse</groupId>
             <artifactId>clickhouse-jdbc</artifactId>
+<!--            本地调试注释此项-->
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-clickhouse/pom.xml]: 删除此处, 无需注释掉

**Comment by Zzm0809** [comment]
请删除无用的注释掉的代码

File: dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java [hunk 6]
```diff
@@ -53,45 +56,70 @@ public class ClickHouseTest {
     private final ClickHouseDriver clickHouseDriver = new ClickHouseDriver();
 
     public Driver getDriver() {
-        DriverConfig<AbstractJdbcConfig> config = new DriverConfig<>();
-        config.setType(clickHouseDriver.getType());
-        config.setName(clickHouseDriver.getName());
-        config.setConnectConfig(
-                AbstractJdbcConfig.builder().ip(IP).port(PORT).url(url).build());
-        return Driver.build(config);
+//        DriverConfig<AbstractJdbcConfig> config = new DriverConfig<>();
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java]: 使用此种方式进行单元测试

Comment (LiuHao0606) [reply dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java]: okay

**Comment by LiuHao0606** [comment]
close

**Pull request closed without merging**